packages/share_plus/share_plus/android/src/main/kotlin/dev/fluttercommunity/plus/share/SharePlusPendingIntent.kt

```diff
@@ -0,0 +1,43 @@
+package dev.fluttercommunity.plus.share
+
+import android.content.*
+import android.os.Build
+
+/**
+ * This helper class allows us to use FLAG_MUTABLE on the PendingIntent used in the Share class,
+ * as it allows us to make the underlying Intent explicit, therefore avoiding any risks an implicit
+ * mutable Intent may carry.
+ *
+ * When the PendingIntent is sent, the system will instantiate this class and call `onReceive` on it.
+ */
+internal class SharePlusPendingIntent: BroadcastReceiver() {
+    /**
+     * Companion object to achieve static behaviour.
```

Comment:
I think this comment is an overkill and doesn't bring value, considering the comment above the var below.